UKHomeOffice/cerberus-service
# COP-10472 - Occupant counts (on update)

## Description
This PR fixes a bug on occupants count where an update has been received.

## To Test
- Pull & run against dev
- Load up this task https://www.dev.cerberus.cop.homeoffice.gov.uk/tasks/DEV-20220721-1472
- Observe the occupant counts between the 3 versions.
- Between the updates, `NaN` should not be rendered when the counts have been updated.
